src/services/codefixes/convertToTypeOnlyExport.ts

```diff
@@ -44,7 +44,11 @@ namespace ts.codefix {
                     exportDeclaration.modifiers,
                     /*isTypeOnly*/ true,
                     exportClause,
-                    exportDeclaration.moduleSpecifier));
+                    exportDeclaration.moduleSpecifier),
+                {

```

Comment:
This particular `changes.replaceNode` is so simple that it really should have been `changes.insertModifierBefore`, which would not only avoid clobbering the comments, but also preserve any other wacky formatting the user had chosen to write in the export declaration. A corresponding test might be converting something like

```ts
export {
  T1,
  T2,
  T3,
};
```

because I’m pretty sure as it is today, that one would get rewritten onto one line and lose its trailing comma.

Reply:
Thanks for the review @andrewbranch. Did you want me to make that change or leave that for another PR?

Reply:
It would be great if we can go ahead and do it here—I’m suggesting it primarily because it’s a better way to solve the comment problem.

Reply:
I went ahead and pushed this change.